core/modules/views/includes/view.inc

```diff
@@ -1869,7 +1869,7 @@ class view {
   function clone_view() {
     $clone = clone($this);
 
-    $keys = array('current_display', 'display_handler', 'build_info', 'built', 'executed', 'attachment_before', 'attachment_after', 'field', 'argument', 'filter', 'sort', 'relationship', 'header', 'footer', 'empty', 'query', 'inited', 'style_plugin', 'plugin_name', 'exposed_data', 'exposed_input', 'exposed_widgets', 'many_to_one_tables', 'feed_icon');
+    $keys = array('current_display', 'display_handler', 'build_info', 'built', 'executed', 'attachment_before', 'attachment_after', 'field', 'argument', 'filter', 'sort', 'relationship', 'header', 'footer', 'empty', 'query', 'inited', 'style_plugin', 'plugin_name', 'exposed_data', 'exposed_input', 'exposed_widgets', 'many_to_one_tables', 'feed_icon', 'module');
```

Comment:
Unsetting the `module` key brings the following error : ```
Error message
Notice: Undefined property: view::$module in view->save() (line 1798 of /path/to/core/modules/views/includes/view.inc).```

`$clone->module` should be set to an empty string right after, as it's done for `attachment_before` and `attachment_after` for example.